app/controllers/posts_controller.rb

```diff
@@ -1,12 +1,39 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts.includes(:comments, :likes)
+    @posts = @user.posts.order(created_at: :desc).includes(:comments, :likes)
   end
 
   def show
-    @post = Post.find(params[:user_id])
+    @post = Post.find(params[:id])
     @user = @post.author
     @comments = @post.comments
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
```

Comment:
When I try to create a new Post , It wasn't created and I got an error `unknown attribute 'author_id' for Post`  Kindly Fix this issue  👍

Reply:
Hi @Danieldotcomcoder 
Thanks for your review but on my side, it is working perfectly.

Here is the screenshot I took for adding new post 

![required](https://user-images.githubusercontent.com/65281462/185676429-c93be2a9-d5e9-4afe-aded-96ada007da2e.png)

Reply:
Hello @Pazzo97 , I tried multiple times it didn't work on my end, I'm sorry for that, I'm still trying to figure what's possibly wrong.

Reply:
@ Can I call you on zoom? for few mins